src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/RGBA.txt

```diff
@@ -0,0 +1,53 @@
+﻿>> RGBA(1, 2, 3, 0.4)
+RGBA(1,2,3,0.4)
+
+>> RGBA(0, 0, 0, 0)
+RGBA(0,0,0,0)
+
+>> RGBA(255, 255, 255, 1.0)
+RGBA(255,255,255,1)
+
+>> RGBA(-1, 255, 255, 1.0)
+Error({Kind:ErrorKind.InvalidArgument})
+
+>> RGBA(1.5, 255, 255, 1.0)
+RGBA(1,255,255,1)
+
+>> RGBA(256, 255, 255, 1.0)
+Error({Kind:ErrorKind.InvalidArgument})
+
+>> RGBA(255, -1, 255, 1.0)
+Error({Kind:ErrorKind.InvalidArgument})
+
+>> RGBA(255, 1.5, 255, 1.0)
+RGBA(255,1,255,1)
+
+>> RGBA(255, 256, 255, 1.0)
+Error({Kind:ErrorKind.InvalidArgument})
+
+>> RGBA(255, 255, -1, 1.0)
+Error({Kind:ErrorKind.InvalidArgument})
+
+>> RGBA(255, 255, 1.5, 1.0)
+RGBA(255,255,1,1)
+
+>> RGBA(255, 255, 256, 1.0)
+Error({Kind:ErrorKind.InvalidArgument})
+
+>> RGBA(255, 255, 255, -0.01)
+Error({Kind:ErrorKind.InvalidArgument})
+
+>> RGBA(255, 255, 255, 1.01)
+Error({Kind:ErrorKind.InvalidArgument})
+
+>> RGBA(Blank(),Blank(),Blank(),1)
+RGBA(0,0,0,1)
+
+>> RGBA(255,255,255,Blank())
+RGBA(255,255,255,0)
+
+>> RGBA(255,255,255,0.003)
+RGBA(255,255,255,0.004)
+
+>> RGBA(255,255,255,0.0003)
+RGBA(255,255,255,0)
```

Comment:
Note these same tests will be run against JScript. 
We should comment whether 
1. this behavior is language semantics  (ie, other backends should do the same thing) 
2. or these are rounding errors (that may be specific to the backend).

Reply:
updated.